app/models/embedded_ansible_worker.rb

```diff
@@ -18,6 +19,7 @@ def start_runner
       end
     end
   end
+  alias_method :start_runner_via_spawn, :start_runner_via_fork
```

Comment:
@carbonin please review this part.  It's kinda icky that this worker needs to know that it has to override both of these methods with a threaded runner but it seems like the simplest solution to get this feature on master